src/Azure.Functions.Cli/StaticResources/function_app.py.template

```diff
@@ -0,0 +1,74 @@
+import azure.functions as func
+
+app = func.FunctionApp()
+
+
+# To run wsgi app using new prog model python function, please uncomment below section and comment out rest of file
```

Comment:
I agree in the general use case, though in the beginning (first few times) it may also be helpful. 

I am wondering - do you think there's value in having some commented out code examples and reference to docs while in preview, and replacing it with just reference to docs when going to GA?

Reply:
Yeah that sounds reasonable to me